csharp/ql/test/query-tests/Likely Bugs/LeapYear/AntiPattern1/Program.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
```

Comment:
Move to ` csharp/ql/test/query-tests/Likely Bugs/UnsafeYearConstruction/Program.cs`